indico/modules/events/management/forms.py

```diff
@@ -61,4 +64,17 @@ def _create_coordinator_priv_fields(cls):
             cls.priv_fields.add(name)
 
 
+class PosterPrintingForm(IndicoForm):
+    template = SelectField(_('Template'))
+    margin_horizontal = FloatField(_('Horizontal margins'), [InputRequired()], default=0)
+    margin_vertical = FloatField(_('Vertical margins'), [InputRequired()], default=0)
+    page_size = IndicoEnumSelectField(_('Page size'), enum=PageSize, default=PageSize.A4)
+
+    def __init__(self, event, **kwargs):
+        all_templates = set(event.designer_templates) | get_inherited_templates(event)
+        poster_templates = [tpl for tpl in all_templates if tpl.type.name == 'poster']
+        super(PosterPrintingForm, self).__init__(**kwargs)
+        self.template.choices = sorted(((unicode(tpl.id), tpl.title) for tpl in poster_templates), key=itemgetter(1))
```

Comment:
I thought `PySequence_List` would be more efficient when it can just copy an existing list of a known size etc, but it looks like it's [not the case][1] and it's literally equivalent on calling `list()` :/

[1]: https://github.com/python/cpython/blob/72dccde884d89586b0cafd990675b7e21720a81f/Objects/abstract.c#L1849